esx_service/cli/vmdkops_admin.py

```diff
@@ -807,20 +807,23 @@ def tenant_ls_headers():
     headers = ['Uuid', 'Name', 'Description', 'Default_datastore', 'VM_list']
     return headers
 
-def generate_vm_list(vms_uuid):
-    """ Generate vm names with given list of vm uuid"""
-    # vms_uuid is a list of vm_uuid
-    # example: vms_uuid=["vm1_uuid", "vm2_uuid"]
-    # the return value is a string like this vm1,vm2
+def generate_vm_list(vm_list):
+    """ Generate vm names with given list of (vm_uuid, vm_name) from db"""
+    # vm_list is a list of (vm_uuid, vm_name) from db
+    # the return value is a comma separated string of VM names like this vm1,vm2
     res = ""
-    for vm_uuid in vms_uuid:
+    for vm_uuid, vm_db_name in vm_list:
```

Comment:
I don't understand the logic here. In your passed in ```vm_list```, name of the vm is already in ```vm_db_name```, then why still need to call vmdk_utils.get_vm_name_by_uuid() to get the vm name from host for every vm in ```vm_list```? Should we  only call vmdk_utils.get_vm_name_by_uuid()  if the vm_name is not available in DB?